crates/tauri/src/ipc/channel.rs

```diff
@@ -180,9 +181,13 @@ impl JavaScriptChannelId {
       }),
       Some(Box::new(move || {
         let current_index = counter_clone.load(Ordering::Relaxed);
-        let _ = webview_clone.eval(format!(
-          "window['_{callback_id}']({{ end: true, index: {current_index} }})",
-        ));
+        let _ = webview_clone.eval(format!("window['_{callback_id}']()",));
```

Comment:
I don't quite understand this change, is this some left over code?

Reply:
This was just a mistake.

Reply:
Then why don't we remove it? `let _ = webview_clone.eval(format!("",));` doesn't make sense either